### DISCUSSION TITLE
Welcome to Code Autopilot

### DISCUSSION THREAD

**Discussion by codeautopilot** [insarlab/MintPy#1287] [Show and tell]
# Hi @insarlab! 👋 

I'm Code Autopilot :robot:

Want me to help you solve a bug, implement a new feature, or answer questions? :point_right: [Open your first task here](https://github.com/insarlab/MintPy/issues/new?title=Task+Tile&body=%2A%2AWhat%2A%2A+-+description+of+what+you+me+to+do%0A+Example%3A+Hey+%40autopilot+implement+a+Google+sign-in+on+my+website.+Make+changes+to+the+front+end+and+the+back+end+of+the+application%0A%0A%2A%2AWhy%2A%2A+-+explain+why+this+is+important%0AExample%3A+I+want+to+allow+users+to+signup+and+login+using+their+Google+account) :point_left:

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).
